Package.swift

```diff
@@ -22,7 +22,9 @@ let package = Package(
             revision: "0c1de7149a39a9ff82d4db66234dec587b30a3ad"),
         .package(
             url: "https://github.com/apple/swift-syntax.git",
-            from: "509.0.0")
+            from: "509.0.0"),
+        .package(url: "https://github.com/attaswift/BigInt.git",
+                 .upToNextMajor(from: "5.1.0")),
```

Comment:
Could you get enough test coverage by using `Int128` from swift-numerics? This is something you could include. It would not be able to expose performance problems in extremely large numbers, but it should allow you to validate most of the algorithm.

Reply:
`DoubleWidth` is (essentially) package-internal to swift-numerics; it's not exported.

Reply:
We could copy the source for DoubleWidth into the test utilities, until such time as we have something either in the stdlib or in a package we are willing to import.

Reply:
To clarify, I don't have strong feelings on this, I'd just like to be sure it's a fully informed decision.  Copy-pasting `DoubleWidth` is mechanically easy but it's against general best practice.

[I did suggest putting `DoubleWidth` into its own package](https://forums.swift.org/t/status-check-int128-uint128/67694/17), so that it can be more cleanly reused, but @stephentyrone discouraged it.

Having use of 128-bit integers would improve the unit test's effective coverage significantly.  It's not enough to provide full confidence, since some bugs by nature are only hit with _much_ larger numbers, but it's fairly close.